### PR TITLE
Update Create a Notebook

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 1]
```diff
@@ -9,101 +9,157 @@ menu:
...
-9. (Optional) [Convert a cell into raw Flux script](#view-and-edit-flux-script-in-a-cell) to view and edit the code.
+## Create a notebook from a preset
 
-### Add a data source cell
+To create a new notebook, do the following:
 
-1. Select **Flux Script** or **Query Builder** as your input, and then select or enter the bucket to pull data from.
-2. Select filters to narrow your data.
-3. Select **Preview** (**CTRL + Enter**) or **Run** in the upper left dropdown menu.
+1. In the navigation menu on the left, click **Books** or **Notebooks**.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:28]:
```suggestion
1. In the navigation menu on the left, click **Notebooks**.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 2]
```diff
@@ -9,101 +9,157 @@ menu:
...
 
-### Add an action cell
+    {{< nav-icon "notebooks" >}}
+2. In the **Notebooks** page, click one of the following options under **Create a Notebook**:
+    - **New Notebook**: includes a [query builder cell](#add-a-data-source-cell), a [validation cell](#add-a-validation-cell), and a [visualization cell](#add-a-visualization-cell).
+    - **Set an Alert**: includes a [query builder cell](#add-a-data-source-cell), a [validation cell](#add-a-validation-cell), a [visualization cell](#add-a-visualization-cell), and an [alert builder cell](#add-an-action-cell).
+    - **Schedule a Task**: includes a [Flux Script editor cell]((#add-a-data-source-cell), a [validation cell](#add-a-validation-cell), and a [task schedule cell](#add-an-action-cell).
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:34]:
```suggestion
    - **Schedule a Task**: includes a [Flux Script editor cell](#add-a-data-source-cell), a [validation cell](#add-a-validation-cell), and a [task schedule cell](#add-an-action-cell).
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 3]
```diff
@@ -9,101 +9,157 @@ menu:
...
+To create a new notebook, do the following:
 
-1. Select **Flux Script** or **Query Builder** as your input, and then select or enter the bucket to pull data from.
-2. Select filters to narrow your data.
-3. Select **Preview** (**CTRL + Enter**) or **Run** in the upper left dropdown menu.
+1. In the navigation menu on the left, click **Books** or **Notebooks**.
 
-### Add an action cell
+    {{< nav-icon "notebooks" >}}
+2. In the **Notebooks** page, click one of the following options under **Create a Notebook**:
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:31]:
```suggestion
2. In the **Notebooks** page, select one of the following options under **Create a Notebook**:
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 4]
```diff
@@ -9,101 +9,157 @@ menu:
...
+3.  Enter a name for your notebook in the **Untitled Notebook** field.
+4. Do the following at the top of the page:
+   - Select your local time zone or UTC.
+   - Choose a time [range](/{{% latest "flux" %}}/stdlib/universe/range/) for your data.
+5. Your notebook should have a **Data Source** cell as the first cell. **Data Source** cells provide data to subsequent cells. The presets (listed in step 2) include either a **Query Builder** or a **Flux Script** as the first cell.
+6. To define your data source query, do one of the following:
+   - If your notebook uses a **Query Builder** cell, select your bucket and any additional filters for your query.
+   - If your notebook uses a **Flux Script** cell, enter or paste a [Flux script](/influxdb/v2.1/query-data/flux/).
+7. Select and click **Preview** (or press **CTRL + Enter**) under the notebook title.
+   InfluxDB displays query results in **Validate the Data** and **Visualize the Result** *without writing data or running actions*.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:46]:
```suggestion
7. {{< oss-only >}}

    Select and click **Preview** (or press **CTRL + Enter**) under the notebook title.
    InfluxDB displays query results in **Validate the Data** and **Visualize the Result** *without writing data or 
    running actions*.
    
    {{< /oss-only >}}
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 5]
```diff
@@ -9,101 +9,157 @@ menu:
...
+8. (Optional) Change your visualization settings with the dropdown menu and the {{< icon "gear" >}} **Configure** button at the top of the **Visualize the Result** cell.
+9. (Optional) Toggle the **Presentation** switch to display visualization cells and hide all other cells.
+10. (Optional) Configure notebook actions (**Alert**, **Task**, or **Output to Bucket**).
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:49]:
```suggestion
10. (Optional) Configure notebook actions {{< oss-only >}}(**Alert**, **Task**, or **Output to Bucket**){{< /oss-only >}}{{< cloud-only >}}(**Alert** or **Task**){{< /cloud-only >}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 6]
```diff
@@ -9,101 +9,157 @@ menu:
...
 - To download results as an annotated CSV file, click the **CSV** button.
-- To export to the dashboard, click **Export to Dashboard**.  
+- To export to the dashboard, click **Export to Dashboard**.
+
+## Add a data source cell
+
+Add a [data source cell](/influxdb/v2.1/notebooks/overview/#data-source) to pull information into your notebook.
+
+To add a data source cell, do the following:
+1. Click the {{< icon "notebook-add-cell" >}}.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:89]:
```suggestion
1. Click {{< icon "notebook-add-cell" >}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 7]
```diff
@@ -9,101 +9,157 @@ menu:
...
+2. Select **Flux Script** or **Query Builder** as your input, and then select or enter the bucket to pull data from.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:90]:
```suggestion
2. Select **{{< caps >}}Flux Script{{< /caps >}}** or **{{< caps >}}Query Builder{{< /caps >}}** as your input, and then select or enter the bucket to pull data from.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 8]
```diff
@@ -9,101 +9,157 @@ menu:
...
+3. Select filters to narrow your data.
+4. Select **Preview** (**CTRL + Enter**) or **Run** in the upper left dropdown menu.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:92]:
```suggestion
4. Select {{< oss-only >}}**Preview** (**CTRL + Enter**) or {{< /oss-only >}}**Run** in the upper left drop-down list.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 9]
```diff
@@ -9,101 +9,157 @@ menu:
...
+
+## Add a validation cell
+
+A validation cell uses the **Table** [visualization type](/influxdb/v2.1/visualize-data/visualization-types/) to display query results from a data source cell.
+
+To add a **Table** visualization cell, do the following:
 
-### Add an action cell
+1. Click the {{< icon "notebook-add-cell" >}}.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:100]:
```suggestion
1. Click {{< icon "notebook-add-cell" >}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 10]
```diff
@@ -9,101 +9,157 @@ menu:
...
+2. Under **Visualization**, click **Table**.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:101]:
```suggestion
2. Under **Visualization**, click **{{< caps >}}Table{{< /caps >}}**.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 11]
```diff
@@ -9,101 +9,157 @@ menu:
...
+
+## Add a visualization cell
+
+Add a visualization cell to render query results as a [Visualization type](/influxdb/v2.1/visualize-data/visualization-types/).
+
+To add a Table visualization cell, do the following:
+
+1. Click the {{< icon "notebook-add-cell" >}}.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:109]:
```suggestion
1. Click {{< icon "notebook-add-cell" >}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 12]
```diff
@@ -9,101 +9,157 @@ menu:
...
+2. Under **Visualization**, select one of the following visualization cell-types:
+
+   - Table
+   - Graph
+   - Note
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:114]:
```suggestion
   - **{{< caps >}}Table{{< /caps >}}**: Display data in tabular format.
   - **{{< caps >}}Graph{{< /caps >}}**: Visualize data using InfluxDB visualizations.
   - **{{< caps >}}Note{{< /caps >}}**: Use Markdown to add notes or other information to your notebook.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 13]
```diff
@@ -9,101 +9,157 @@ menu:
...
+
+To modify a visualization cell, see [use visualization cells](#use-visualization-cells).
+For detail on available visualization types and how to use them, see [Visualization types](/influxdb/v2.1/visualize-data/visualization-types/).
+
+## Add an action cell
+
+Add an [action cell](/influxdb/v2.1/notebooks/overview/#action) to filter and apply changes to your data, for example, create an [alert](/influxdb/v2.1/monitor-alert/), process data with a [task](/influxdb/v2.1/process-data/manage-tasks/), or output data to a bucket.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:121]:
```suggestion
Add an [action cell](/influxdb/v2.1/notebooks/overview/#action) to create an [alert](/influxdb/v2.1/monitor-alert/)
{{< cloud-only >}}or{{< /cloud-only >}}{{< oss-only >}},{{< /oss-only >}} process data with a [task](/influxdb/v2.1/process-data/manage-tasks/)
{{< oss-only >}}, or output data to a bucket{{< /oss-only >}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 14]
```diff
@@ -9,101 +9,157 @@ menu:
...
+
+{{% warn %}}
+If your cell contains a custom script that uses any output function to write data to InfluxDB (for example: the `to()` function) or sends data to a third-party service, clicking Preview will write data.
+{{% /warn %}}
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:125]:
```suggestion
{{< oss-only >}}

{{% warn %}}
If your cell contains a custom script that uses any output function to write data to InfluxDB (for example: the `to()` function) or sends data to a third-party service, clicking Preview will write data.
{{% /warn %}}

{{< /oss-only >}}
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 15]
```diff
@@ -9,101 +9,157 @@ menu:
...
 
-Select one of the following action cell-types:
+- [Add an Alert cell](#add-an-alert-cell)
+- [Add an Output to Bucket cell](#add-an-output-to-bucket-cell)
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:128]:
```suggestion
- {{< oss-only >}}[Add an Output to Bucket cell](#add-an-output-to-bucket-cell){{< /oss-only >}}
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 16]
```diff
@@ -9,101 +9,157 @@ menu:
...
 
-#### Alert
+To add an [alert](/influxdb/v2.1/monitor-alert/) to your notebook, do the following:
 
 1. Enter a time range to automatically check the data and enter your query offset.
 2. Customize the conditions to send an alert.
 3. Select an endpoint to receive an alert:
    - Slack and a Slack Channel
    - HTTP post
    - Pager Duty
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:140]:
```suggestion
   - PagerDuty
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 17]
```diff
@@ -9,101 +9,157 @@ menu:
...
-4. (Optional) Personalize your message. By default, the message is "${strings.title(v: r._type)} for ${r._source_measurement} triggered at ${time(v: r._source_timestamp)}!"
-5. Click **Export as Task** to create your alert.
+4. (Optional) Personalize your message. By default, the message is
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:141]:
```suggestion
4. (Optional) Personalize your message. By default, the message is:
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 18]
```diff
@@ -9,101 +9,157 @@ menu:
...
+   ```
+   ${strings.title(v: r._type)} for ${r._source_measurement} triggered at ${time(v: r._source_timestamp)}!
+   ```
+5. Click **Test Alert** to send a test message to your configured **Endpoint**. The test will not schedule the new alert.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:145]:
```suggestion
5. Click **{{< caps >}}Test Alert{{< /caps >}}** to send a test message to your configured **Endpoint**. The test will not schedule the new alert.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 19]
```diff
@@ -9,101 +9,157 @@ menu:
...
+6. Click **Export Alert Task** to create your alert.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:146]:
```suggestion
6. Click **{{< icon "export" >}} {{< caps >}}Export Alert Task{{< /caps >}}** to create your alert.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 20]
```diff
@@ -9,101 +9,157 @@ menu:
...
 
-#### Output to bucket
+### Add an Output to Bucket cell
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:148]:
```suggestion
{{< oss-only >}}
### Add an Output to Bucket cell
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 21]
```diff
@@ -9,101 +9,157 @@ menu:
...
-1. Select a bucket.
-2. Click **Preview** to see what would be written to the bucket without commiting, or click **Run** in the upper left to write, or select **Export as Task** to schedule your output as a task.
+To write **Data Source** results to a bucket, do the following:
 
-#### Task
+1. Click the {{% icon "notebook-add-cell" %}}.
+2. Click **Output to Bucket**.
+3. In the **Choose a bucket** dropdown menu, select or create a bucket.
+4. Click **Preview** to view the query result in validation cells.
+5. Select and click **Run** in the upper left to write the query result to the bucket.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:156]:
```suggestion
5. Select and click **Run** in the upper left to write the query result to the bucket.

{{< /oss-only >}}
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 22]
```diff
@@ -9,101 +9,157 @@ menu:
...
-#### Output to bucket
+### Add an Output to Bucket cell
 
-1. Select a bucket.
-2. Click **Preview** to see what would be written to the bucket without commiting, or click **Run** in the upper left to write, or select **Export as Task** to schedule your output as a task.
+To write **Data Source** results to a bucket, do the following:
 
-#### Task
+1. Click the {{% icon "notebook-add-cell" %}}.
+2. Click **Output to Bucket**.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:153]:
```suggestion
2. Click **{{< caps >}}Output to Bucket{{< /caps >}}**.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 23]
```diff
@@ -9,101 +9,157 @@ menu:
...
+3. In the **Choose a bucket** dropdown menu, select or create a bucket.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:154]:
```suggestion
3. In the **{{< icon "bucket" >}} Choose a bucket** drop-down list, select or create a bucket.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 24]
```diff
@@ -9,101 +9,157 @@ menu:
...
+4. Do the following at the top of the page:
+   - Select your local time zone or UTC.
+   - Choose a time [range](/{{% latest "flux" %}}/stdlib/universe/range/) for your data.
+5. Your notebook should have a **Data Source** cell as the first cell. **Data Source** cells provide data to subsequent cells. The presets (listed in step 2) include either a **Query Builder** or a **Flux Script** as the first cell.
+6. To define your data source query, do one of the following:
+   - If your notebook uses a **Query Builder** cell, select your bucket and any additional filters for your query.
+   - If your notebook uses a **Flux Script** cell, enter or paste a [Flux script](/influxdb/v2.1/query-data/flux/).
+7. Select and click **Preview** (or press **CTRL + Enter**) under the notebook title.
+   InfluxDB displays query results in **Validate the Data** and **Visualize the Result** *without writing data or running actions*.
+8. (Optional) Change your visualization settings with the dropdown menu and the {{< icon "gear" >}} **Configure** button at the top of the **Visualize the Result** cell.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:47]:
```suggestion
8. (Optional) Change your visualization settings with the drop-down menu and the {{< icon "gear" >}} **Configure** button at the top of the **Visualize the Result** cell.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 25]
```diff
@@ -9,101 +9,157 @@ menu:
...
-- Graph
-- Note
-For detail on available visualization types and how to use them, see [Visualization types](/influxdb/cloud/visualize-data/visualization-types/).
+2. View and edit the Flux script as needed.
 
-#### Visualization
+## Use visualization cells
 
-- To change your visualization type, select a new type from the dropdown menu at the top of the cell.
+- To change your [visualization type](/influxdb/v2.1/visualize-data/visualization-types/), select a new type from the dropdown menu at the top of the cell.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:78]:
```suggestion
- To change your [visualization type](/influxdb/v2.1/visualize-data/visualization-types/), select a new type from the drop-down list at the top of the cell.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 26]
```diff
@@ -9,101 +9,157 @@ menu:
...
 
-#### Output to bucket
+### Add an Output to Bucket cell
 
-1. Select a bucket.
-2. Click **Preview** to see what would be written to the bucket without commiting, or click **Run** in the upper left to write, or select **Export as Task** to schedule your output as a task.
+To write **Data Source** results to a bucket, do the following:
 
-#### Task
+1. Click the {{% icon "notebook-add-cell" %}}.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:152]:
```suggestion
1. Click {{% icon "notebook-add-cell" %}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 27]
```diff
@@ -9,101 +9,157 @@ menu:
...
-### View and edit Flux script in a cell
-Convert your notebook cells into raw Flux script to view and edit the code. Conversion to Flux is available for the following cell types:
-  - **Inputs: Metric Selector**
-  - **Transform: Column Editor** and **Downsample**
+To add a [task](/influxdb/v2.1/process-data/manage-tasks/) to your notebook, do the following:
 
-1. Click the overflow menu icon in the cell you want to view as Flux, and then select **Convert to |> Flux**. You won't be able to undo this step.
-    A Flux cell appears with underlying script for the selected cell.
-3. View and edit the Flux script as needed.
+1. Click the {{% icon "notebook-add-cell" %}}.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:162]:
```suggestion
1. Click {{% icon "notebook-add-cell" %}}.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 28]
```diff
@@ -9,101 +9,157 @@ menu:
...
+2. Click **Task**.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:163]:
```suggestion
2. Click **{{< caps >}}Task{{< /caps >}}**.
```

File: content/influxdb/v2.1/notebooks/create-notebook.md [hunk 29]
```diff
@@ -9,101 +9,157 @@ menu:
...
+3. Enter a time and an offset to schedule the task.
+4. Click **Export as Task** to save.
```
Comment (sanderson) [inline content/influxdb/v2.1/notebooks/create-notebook.md:165]:
```suggestion
4. Click **{{< icon "task" >}} {{< caps >}}Export as Task{{< /caps >}}** to save.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.1/notebooks/create-notebook.md (hunk 16 above)
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/create-notebook.md:151]: I submitted a fix for this to the UI repo (https://github.com/influxdata/ui/pull/3335). "PagerDuty" is the registered trademark name.

File: content/influxdb/v2.1/notebooks/create-notebook.md (hunk 28 above)
Comment (jstirnaman) [reply content/influxdb/v2.1/notebooks/create-notebook.md]: v2.1 buttons aren't uppercase yet.

File: content/influxdb/v2.1/notebooks/create-notebook.md (hunk 29 above)
Comment (jstirnaman) [reply content/influxdb/v2.1/notebooks/create-notebook.md]: v2.1 buttons aren't uppercase yet.

File: content/influxdb/v2.1/notebooks/create-notebook.md (hunk 16 above)
Comment (sanderson) [reply content/influxdb/v2.1/notebooks/create-notebook.md:151]: This change just got merged into the UI repo, so we're probably good to make it here and future-proof it.

**Pull request merged**